Adress cmake linter problems

Blast545 opened the pull request:
This should close #31 

This should address the cmake warnings in the performance jobs of the buildfarm.
To be backported to galactic after merging.

Reference:
https://build.ros2.org/view/Gci/job/Gci__nightly-performance_ubuntu_focal_amd64/99/#showFailuresLink
https://build.ros2.org/view/Gci/job/Gci__nightly-performance_ubuntu_focal_amd64/99/testReport/junit/performance_test/lint_cmake/linelength__src_idlgen_opendds_CMakeLists_txt_102_/

Tagging @cottsay as assigned and @clalancette for awareness.

Signed-off-by: Jorge Perez <jjperez@ekumenlabs.com>

cottsay commented:
Two things:
1. Please submit these changes upstream to https://gitlab.com/ApexAI/performance_test/
2. We're using the `run_on_buildfarm` branch on the buildfarm, not `master`

Once the PR is open upstream, we can consider merging this early in anticipation of them accepting the changes.

Blast545 commented:
1. The `upstream` repository doesn't have the portion of the code affected by this patch.
https://gitlab.com/ApexAI/performance_test/-/tree/master/performance_test/src
I don't think there's a way to open a PR with this there, please correct me if I'm missing something.

2. I rebased this locally and at some point I deleted the branch and now the UI think this was merged when it wasn't. I re opened this PR targetting `run_on_buildfarm` on #38.
